libr/debug/p/native/maps/windows.c

```diff
@@ -54,61 +62,66 @@ static RDebugMap *add_map(RList *list, const char *name, ut64 addr, ut64 len, ME
 	default:
 		perm = 0;
 	}
-	map_name = r_str_newf ("%-8s %s", map_type, name);
+	char *map_name = r_str_newf ("%-8s %s", map_type, name);
 	if (!map_name) {
 		perror ("r_str_newf");
```

Comment:
no need to perror for a string, drop this line

Reply:
if this string is used only for 1 call and is then freed you can just use `sdb_fmt` and skip the free below

Reply:
I'm not certain about its usage because I didn't actually write that. I've only fixed styles and bugs here and there if I saw any.